package examples.exceptions;

import examples.exceptions.*;

public class Assignment1 {
	
	public void FactorialTest() {
		try {
			long out = MyLibrary.Fac(5);
			System.out.format("5 factorial is %d \n", MyLibrary.Fac(5));
		}
		catch (NotAValidIntegerException nex) {
			nex.getMessage();
		}
		finally {
			System.out.format("5 factorial is %d \n", MyLibrary.Fac(5));
		}
	}

	public static void main(String[] args) {
		// TODO Auto-generated method stub
		MyLibrary ml = new MyLibrary();
		
		
		
		
	}

}
